alpine/Dockerfile

```diff
@@ -48,7 +49,10 @@ RUN set -x \
 		--build="$gnuArch" \
 		--enable-sasl \
 		--enable-sasl-pwdb \
+		--enable-tls \
```

Comment:
Frankly, I'd rather just not enable this on Alpine until the issues building it are resolved upstream:

```suggestion
# TODO enable TLS here after https://github.com/memcached/memcached/issues/572 is resolved
#		--enable-tls \
```